MaeYoungPhan/md-bay-game-server
# Bay items view

# Description of Proposed Changes

added bayitems to views. moved property decorator to bay items model from gamer model. custom actions for find and reset included in bayitems_view

---


# Steps to Test

[] Fetch branch and switch to it, run debugger
[] Open Postman
[] GET request to http://localhost:8000/bayitems. RESPONSE BODY: List of dictionaries of bay items STATUS: 200
[] GET request to http://localhost:8000/bayitems/2. RESPONSE BODY: dictionary for item 2 STATUS: 200
[] GET request to http://localhost:8000/bayitems/20. RESPONSE BODY: blank STATUS: 404
[] POST request to http://localhost:8000/bayitems/2/find. RESPONSE BODY: "item found" STATUS: 201
[] GET request to http://localhost:8000/bayitems/2. RESPONSE BODY: dictionary for item 2 - verify logged in gamer pk is included in gamers list STATUS: 200
[] DELETE request to http://localhost:8000/bayitems/2/reset. RESPONSE BODY: none STATUS: 204
[] Proofread files changed
